src/main/java/dk/kb/labsapi/ImageExport.java

```diff
@@ -384,7 +391,7 @@ public Stream<IllustrationMetadata> getMetadataForIllustrations(SolrDocument doc
      * @param metadataMap which delivers overall information on the export.
      * @param exportFormat determines what kind of export that are to be done.
      */
-    public void createZipOfImages(Stream<? extends BasicMetadata> imageMetadata, OutputStream output, Map<String, Object> metadataMap, String exportFormat) throws IOException {
+    public AtomicInteger createZipOfImages(Stream<? extends BasicMetadata> imageMetadata, OutputStream output, Map<String, Object> metadataMap, String exportFormat) throws IOException {
```

Comment:
While an `AtomicInteger` is needed for doing the counting in `exportImage`, there is no reason to return that. Please return a plain `int` instead.